src/Mod/Path/Path/Op/Helix.py

```diff
@@ -198,26 +200,6 @@ def circularHoleExecute(self, obj, holes):
             endPoint = FreeCAD.Vector(hole["x"], hole["y"], obj.FinalDepth.Value)
             args["edge"] = Part.makeLine(startPoint, endPoint)
 
-            # move to starting position
-            self.commandlist.append(
-                Path.Command("G0", {"Z": obj.ClearanceHeight.Value})
-            )
-            self.commandlist.append(
-                Path.Command(
-                    "G0",
-                    {
-                        "X": startPoint.x,
-                        "Y": startPoint.y,
-                        "Z": obj.ClearanceHeight.Value,
-                    },
-                )
-            )
-            self.commandlist.append(
-                Path.Command(
-                    "G0", {"X": startPoint.x, "Y": startPoint.y, "Z": startPoint.z}
-                )
-            )
-
```

Comment:
Isn't the move to the `startPoint` still required?

Reply:
This was replaced with new code in .../Path/Base/Generator/helix.py .

`startPoint` is just the hole center and not there the operation should really start.